clang/include/clang/Basic/Builtins.td

```diff
@@ -660,15 +660,23 @@ def Clz : Builtin, BitShort_Int_Long_LongLongTemplate {
   let Prototype = "int(unsigned T)";
 }
 
-// FIXME: Add int clzimax(uintmax_t)
+def Clzg : Builtin {
+  let Spellings = ["__builtin_clzg"];
+  let Attributes = [NoThrow, Const, CustomTypeChecking];
```

Comment:
Should `Constexpr` be in this list of attributes?

https://godbolt.org/z/Efd3EW5xE

Reply:
I will implement `constexpr` support. It requires adding a few lines in clang/lib/AST/ExprConstant.cpp and clang/lib/AST/Interp/InterpBuiltin.cpp.

Reply:
Filed https://github.com/llvm/llvm-project/issues/86549